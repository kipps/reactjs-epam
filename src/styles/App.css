h1 {
    color: #27aedb;
}

div.header {
    padding: 20px;
    text-align: center;
}

.text-center {
    text-align: center;
}

.testBlock {
    margin-bottom: 16px;
}
.testBlock > div {
    border: 1px solid rgba(0,0,0,.1);
    border-radius: 3px;
    text-align: center;
    padding: 20px;
}