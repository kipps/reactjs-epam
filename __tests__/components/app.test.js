import React from 'react';
import { shallow } from 'enzyme/build';
import { configure } from 'enzyme/build';
import renderer from 'react-test-renderer';
import Adapter from 'enzyme-adapter-react-16/build';

import App from "../../src/container/app";

configure({ adapter: new Adapter() });

describe("Search Result ", () => {
  it("renders default App", () =>
    expect(
      shallow(<App />)
        .find('div.App')
        .length
    ).toBe(1));

  it('renders App correctly', () => {
    const tree = renderer
      .create(<App />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});