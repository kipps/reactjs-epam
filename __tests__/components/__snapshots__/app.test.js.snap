// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search Result  renders App correctly 1`] = `
<div
  className="App body"
>
  <header
    className="Header pt-24 pb-24"
  >
    <div
      className="container"
    >
      <div
        className="mb-24 row"
      >
        <div
          className="flex v-center col"
        >
          <h1
            className="white f-large"
          >
            Movies for fun
          </h1>
        </div>
        <div
          className="flex v-center h-right col"
        >
          <div
            className="User"
          >
            <a
              className="User__link black"
              href="#"
            >
              Roman Shevchenko
            </a>
          </div>
        </div>
      </div>
      <div
        className="row"
      >
        <div
          className="col"
        >
          <input
            className="full"
            placeholder="Finde your movie"
            type="search"
          />
          <div
            className="flex space-between v-center pt-8"
          >
            <div
              className="flex v-center"
            >
              <span>
                Search by:
              </span>
              <div
                className="ml-8 Radio"
              >
                <label
                  className="inline ml-8 mr-8"
                  name="sort"
                >
                  <span
                    className="inline mr-4"
                  >
                    Genre
                  </span>
                  <input
                    name="sort"
                    type="radio"
                  />
                </label>
                <label
                  className="inline ml-8 mr-8"
                  name="sort"
                >
                  <span
                    className="inline mr-4"
                  >
                    Title
                  </span>
                  <input
                    name="sort"
                    type="radio"
                  />
                </label>
                <label
                  className="inline ml-8 mr-8"
                  name="sort"
                >
                  <span
                    className="inline mr-4"
                  >
                    Other
                  </span>
                  <input
                    name="sort"
                    type="radio"
                  />
                </label>
              </div>
            </div>
            <div>
              <button
                className="btn btn-danger btn-sm"
                disabled={false}
                onClick={[Function]}
                type="button"
              >
                Search
              </button>
            </div>
          </div>
        </div>
      </div>
    </div>
  </header>
  <div
    className="Content"
  >
    <div
      className="pt-8 pb-8 grey"
    >
      <div
        className="container"
      >
        <div
          className="row"
        >
          <div
            className="flex v-center space-between col"
          >
            <p>
              <span>
                <b>
                  0
                </b>
                 movies found
              </span>
            </p>
            <p>
              <span
                className="inline mr-8"
              >
                Sort by 
                <b>
                  Release date
                </b>
              </span>
              |
              <span
                className="red inline ml-8"
              >
                Rating ___
              </span>
            </p>
          </div>
        </div>
      </div>
    </div>
    <div
      className="pt-24 pb-24"
    >
      <div
        className="container"
      >
        <div
          className="row"
        >
          <div
            className="mb-24 mr-16 ml-16"
          >
            <div
              className="Movie"
            >
              <img
                className="Movie__img"
                src="http://localhost/src/images/medium.jpg"
              />
              <div>
                <div
                  className="flex space-between v-center mb-8"
                >
                  <h3>
                    Begin from start
                  </h3>
                  <span
                    className="Movie__date"
                  >
                    2018
                  </span>
                </div>
                <span>
                  <small>
                    Драма, Комедия
                  </small>
                </span>
              </div>
            </div>
          </div>
          <div
            className="mb-24 mr-16 ml-16"
          >
            <div
              className="Movie"
            >
              <img
                className="Movie__img"
                src="http://localhost/src/images/medium_0.jpg"
              />
              <div>
                <div
                  className="flex space-between v-center mb-8"
                >
                  <h3>
                    Outlaw king
                  </h3>
                  <span
                    className="Movie__date"
                  >
                    2017
                  </span>
                </div>
                <span>
                  <small>
                    Драма, Комедия
                  </small>
                </span>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <footer
    className="Footer"
  >
    <p>
      epam's ReactJS course
    </p>
  </footer>
</div>
`;
